code/modules/vehicles/multitile/multitile_bump.dm

```diff
@@ -469,14 +478,15 @@
 			KnockDown(0.5, 1)
 		else
 			KnockDown(1, 1)
-
 	else if(V.vehicle_flags & VEHICLE_CLASS_LIGHT)
+		dmg = TRUE
 		if(get_target_lock(driver.faction))
 			KnockDown(0.5, 1)
+			apply_damage(5 + rand(0, 5), BRUTE)
+			to_chat(V.seats[VEHICLE_DRIVER], SPAN_WARNING(SPAN_BOLD("*YOU RAMMED AN ALLY AND HURT THEM!*")))
```

Comment:
```suggestion
			to_chat(V.seats[VEHICLE_DRIVER], SPAN_WARNING(SPAN_BOLD("YOU RAMMED AN ALLY AND HURT THEM!")))
```
idk if the allcaps is 100% needed

Reply:
As I found out, DANGER style is considered combat so if you have combat chatlogs disabled, you won't see them, otherwise I'd made it in DANGER style and no all-caps.